BDArmory/Weapons/Missiles/MultiMissileLauncher.cs

```diff
@@ -779,6 +779,11 @@ IEnumerator salvoFire(bool LaunchThenDestroy)
                 }
                 ml.launched = true;
                 ml.TargetPosition = vessel.ReferenceTransform.position + (vessel.ReferenceTransform.up * 5000); //set initial target position so if no target update, missileBase will count a miss if it nears this point or is flying post-thrust
+                if (ml == null)
```

Comment:
Similarly, this isn't needed here with the check directly after the `yield return ...` further up that I added.